### ISSUE TITLE
Carthage: "no shared framework schemes"

### ISSUE THREAD

**Issue by zdnk** [appspector/ios-sdk#10]
This pops up when trying to install using carthage.

**Comment by deszip** [comment]
Thanks for reporting, we'll check this

**Comment by zdnk** [comment]
Well, you need to publish file like this:
`appspector.json`
```
{
    "1.2.2": "https://github.com/appspector/ios-sdk/raw/1.2.2/AppSpectorSDK.framework.zip"
}
```

and to installation guide put something like
`Cartfile`
```
binary "https://github.com/appspector/ios-sdk/raw/1.2.2/appspector.json"
```

Please check Carthage documentation for more information.
I currently made this workaround locally.

**Comment by deszip** [comment]
We have released 1.2.3 version today, and it fixes your issue, we now have AppSpector.json file in public repo. Can you please verify it works? Thanks.

**Comment by zdnk** [comment]
Shouldnt the path to .json file be rather on master branch? I guess you wont get any updates if you use the 1.2.3 tag :)

**Comment by zdnk** [comment]
I am also getting this error:
```
dyld: Library not loaded: @rpath/openssl.framework/openssl
  Referenced from: ..../Frameworks/AppSpectorSDK.framework/AppSpectorSDK
  Reason: image not found
```

**Comment by deszip** [comment]
You right we can use branch name instead of a tag but I think using tags in path makes support easier.
Error with openssl is already fixed in 1.2.4 released today. Could you please give it a try?

**Comment by deszip** [comment]
Closing this, feel free to reopen if issue still exists.
